yarn.lock

```diff
@@ -5739,7 +5943,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@react-native-community/cli-platform-ios@npm:12.3.7, @react-native-community/cli-platform-ios@npm:^12.1.1":
+"@react-native-community/cli-platform-ios@npm:12.3.7":
```

Comment:
There are some old packages in here still. Probably because of the win32 packages.